### DISCUSSION TITLE
Customize Footer

### DISCUSSION THREAD

**Discussion by iqlal** [cotes2020/jekyll-theme-chirpy#1273] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

How to overwrite existing footer / change footer ?

### Operations you have already tried

footer file not found in source code (I use starter)

**Comment by pseudology** [comment]
found a fix? facing the same trouble

**Comment by DmitriyFrogo** [comment]
Guys, if you need to change something that not included in chirpy-starter, just download these files from that repo and place it in same directory, than change what you needed

**Reply by DmitriyFrogo** [reply, marked as answer]
If you want to change something in footer, than you need to download /_includes/footer.html from this repo and place it in /_includes/ folder

**Reply by pseudology** [reply]
i tried but i got some kind of error when building the site

**Reply by pseudology** [reply]
nvm i had the wrong version of chirpy. Thanks!
